src/data/collection.json

```diff
@@ -2826,5 +2826,21 @@
 		"author": "adamtlangley",
 		"notes": "Target practice for ffuf",
 		"badge": "adamtlangley/ffufme"
+	},
+	{
+		"url": "https://pentest-ground.com/",
+		"name": "Pentest-Ground",
+		"collection": [
+			"online",
+			"container"
```

Comment:
Is the container available to users to deploy themselves? If not you should drop this and just leave online.